Automattic/wp-calypso
# RT: Hide Get Started CTA from the navbar on the details page

Related to https://github.com/Automattic/dotcom-forge/issues/8600

## Proposed Changes

* Hide the `Get Started` CTA from the navbar on the readymade template details page, since it can distract users away from the RT `Pick this layout` CTA

|Before|After|
|---|---|
|![image](https://github.com/user-attachments/assets/00e841bf-ebab-42ce-8232-4f0284adc4f6)|![image](https://github.com/user-attachments/assets/29e3bdf8-2b4d-4868-ab47-ba5b1720890c)|

## Testing Instructions



* Open the live preview
* Go to `/patterns`
* Scroll down to the site layouts
* Click on one of the layouts
* Check that the details page does not have a `Get started` button on the top right corner of the navbar

## Pre-merge Checklist
